src/devices/heiman.ts

```diff
@@ -301,6 +301,7 @@ export const definitions: DefinitionWithExtend[] = [
             await endpoint.read("ssIasWd", ["maxDuration"]);
         },
         exposes: [
+            e.binary("siren_state", ea.STATE, "ON", "OFF").withDescription("Alarm state (Virtual)"),
```

Comment:
This can be added to every device having `tz.warning`, maybe good to add it to `exposes.ts`